OpenRA.Mods.Common/Traits/BotModules/Squads/States/AirStates.cs

```diff
@@ -81,6 +92,13 @@ protected static Actor FindDefenselessTarget(Squad owner)
 						return pos;
 					}
 				}
+
+				// If the next iteration will be beyond the maximum, reset to 0,0 and continue scanning until we're back to the starting coordinates.
```

Comment:
Maybe I'm reading something wrong here, but it looks like this only resets to `0,startY`? It is probably better approach to set `startY` to 0 as well as `i`, and before that set `maxX` to `startX` and `maxY` to `startY` and remove `scanReset`.

Reply:
Good shout, though I think I need to keep `scanReset` to stop it infinitely looping (`i` would never become greater than `maxX`). Setting `maxX/Y` to `startX/Y` as suggested removes the need for the `scanReset` check in the inner loop too, which is a bonus.